iai_boxy_base_2/robots/base.URDF.xacro

```diff
@@ -1 +0,0 @@
-base.URDF
```

Comment:
Why did you delete this file?

Reply:
Because every files are xacro, I want to make a base file that conclude option (with or without odom). So it must be a xacro

Reply:
Have you made sure that this file is not used anywhere in this repo?

Reply:
I just checked it, this file is not used anywhere else